Integrations/CounterTack/CounterTack.py

```diff
@@ -0,0 +1,1411 @@
+    """
+    request specific file information
+    """
+    suffix_url = 'endpoints/files/' + file_id
+    response = http_request('GET', suffix_url)
+
+    return response
+
+
+def get_file_information():
+
+    """
+    Get the information of a specific file
+
+    demisto parameter: (string) file_id
+        The unique ID of the extracted file
+    """
+    context = {}
+    files_standards = []
+    file_id = demisto.args().get('file_id')
+    response = file_information_request(file_id)
+
+    data = {
+        'endpoint_name': response.get('endpoint_name'),
+        'path': response.get('path'),
+        'size': response.get('size'),
+        'extraction_time': response.get('extraction_time'),
+        'status': response.get('status'),
+    }
+
+    files_standards.append({
+            'Size': response.get('size'),
+            'MD5': response.get('md5'),
+            'SHA256': response.get('sha256'),
+            'SSDeep': response.get('ssdeep'),
+            'Path': response.get('path')
+    })
+
+    context['CounterTack.File(val.Id && val.Id === obj.Id)'] = createContext(response, keyTransform=underscoreToCamelCase)
+    context[outputPaths['file']] = files_standards
+    headers = ['endpoint_name', 'path', 'size', 'status', 'extraction_time']
+    entry = {
+        'Type': entryTypes['note'],
+        'Contents': response,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('CounterTack File Information:', data, headers, removeNull=True),
+        'EntryContext': context
+    }
+    demisto.results(entry)
+
+
+def download_file_request(file_id):
+
+    #This request downloads an extracted file.
+
+    suffix_url = 'downloads/extractedfiles/' + file_id
+    response = http_request('GET', suffix_url)
+    return response
+
+
+def download_file():
+    """
+    Download an extracted file in a ZIP format.
+
+    demisto parameter: (string) file_id
+        The unique ID of the extracted file
+    """
+
+    file_id = demisto.args().get('file_id')
+    response = download_file_request(file_id)
+
+    demisto.results(fileResult(file_id + '.zip', response.content))
+
+
+"""
+
+BEHAVIORS
+
+"""
+
+
+def get_behaviors_request():
+    """
+    This request retrieves information on a collection of behaviors.
+    """
+    suffix_url = 'behaviors'
+
+    response = http_request('GET', suffix_url)
+    return response
+
+
+def get_behaviors():
+    """
+    retrieve information on a collection of behaviors.
+    """
+    data = []
+    behaviors = get_behaviors_request()
+    for behavior in behaviors:
+        data.append({
+            'Id': behavior.get('id'),
+            'Name': behavior.get('name'),
+            'Type': behavior.get('type'),
+            'Impact_Level': behavior.get('impact_level'),
+            'lastReported': behavior.get('last_reported'),
+            'Endpoint_ID': behavior.get('endpoint_id')
+        })
+
+    context = {
```

Comment:
the context does not match in 100% to the `get-behavior` command, e.g. `EndpointID != EndpointId`
please make sure they fit